ansible/roles_ocp_workloads/ocp4_workload_paloalto_prismacloud/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+silent: false
+
+num_users: "100"
+
+# OpenShift project which contains the Prisma Cloud integration workloads
+ocp4_workload_prisma_project: "twistlock"
+ocp4_workload_prisma_admin_user: "admin"
+ocp4_workload_prisma_sockshop_url: "https://raw.githubusercontent.com/mosuke5/microservices-demo-openshift/master/complete-demo.yaml"
```

Comment:
No roles in AgnosticD may point to an external resource that is not managed by an organizational group. No references to personal repositories are allowed. Perhaps a redhat-cop repository would be appropriate?
https://github.com/redhat-cop/org/issues/new?template=repository-management.md

Reply:
Understood @jkupferer, thanks. I will add this yaml directly to the workload.

Reply:
done!